pkg/cosign/verify.go

```diff
@@ -598,31 +602,32 @@ func verifySignatures(ctx context.Context, sigs oci.Signatures, h v1.Hash, co *C
 		}
 	}
 
-	validationErrs := make([]string, len(sl))
 	signatures := make([]oci.Signature, len(sl))
 	bundlesVerified := make([]bool, len(sl))
 
-	var wg sync.WaitGroup
-
+	t := throttler.New(co.MaxWorkers, len(sl))
```

Comment:
I think we should still have a default, because otherwise, anyone calling the verify API will be required to set it now.

Reply:
addressed,ptal